im-commons/src/main/java/com/wipro/ats/bdre/ResolvePath.java

```diff
@@ -42,9 +42,9 @@ public static String replaceVars(String line) {
         return sb.toString();
     }
 
-    private static String resolveCode(String code) {
-        ;
-        switch (code) {
+    private static String resolveCode(String argCode) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "resolveCode" is 26 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)